api/v1alpha1/shipwrightbuild_types.go

```diff
@@ -8,29 +8,20 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
-
-// ShipwrightBuildSpec defines the desired state of ShipwrightBuild
+// ShipwrightBuildSpec defines the configuration of a Shipwright Build deployment.
 type ShipwrightBuildSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
-
-	// Foo is an example field of ShipwrightBuild. Edit ShipwrightBuild_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	// TargetNamespace is the target namespace where Shipwright's build controller will be deployed.
+	TargetNamespace string `json:"targetNamespace,omitempty"`
```

Comment:
Can this just be `Namespace`?

Reply:
`targetNamespace` is a common convention for operator configuration custom resources. The operator itself can be installed in a different namespace - for example, upstream with OLM operators will get installed by default in the `olm-operators` namespace. The community has converged on "target namespace" to indicate where cluster-scoped operand components are created.

This is different from, say, a MySQL operator that could provision multiple MySQL databases in multiple namespaces. For those the configuration CR would be namespace scoped, and the operands would be the individual MySQL DBs.